### DISCUSSION TITLE
Restore functionality to fully retract tab sidebars

### DISCUSSION THREAD

**Discussion by Lan0s** [zen-browser/desktop#3294] [General]
This feature, which no longer seems to exist in the latest version of Zen (1.0.1-a.20), was a real asset in maximizing on-screen legibility. I think it's a shame it's gone. 

As a reminder: it allowed the tab bar on the left of the screen to be almost completely retracted. This left only the icons of open tabs displayed.

**Comment by 12th-devs** [comment]
@LanceStark, update to a.22 he has brought this back!

**Reply by vraravam** [reply]
a22 is only a re-numbering of a19 - which had the retractable/expand-on-hover behavior of the sidebar.
If you look at the twilight builds (which reflects the latest master branch), the retractable sidebar is not there. So, the euphoria is short-lived imho.

**Reply by 12th-devs** [reply]
@vraravam Yes, but the dev said on Reddit that on his builds he found a way to maintain it so it will be brought back soon!

**Reply by vraravam** [reply]
oh! That would be great! Could you please share the link if you can find it? Would love to confirm this happy news!

**Reply by vraravam** [reply]
(though it still doesn't show up in the twilight builds)

**Reply by 12th-devs** [reply]
@vraravam here is the threads, it should be added back into Twilight soon. It is in his personal nightly builds to my understanding.

https://www.reddit.com/r/zen_browser/comments/1h1lf6t/comment/lzfpmuj/

https://www.reddit.com/r/zen_browser/comments/1h22yc2/comment/lzg4tdu/

**Reply by Lan0s** [reply]
Thanks lads !!
